test/configCases/defer-import/attribute-error/index.js

```diff
@@ -0,0 +1,11 @@
+import { f } from "./mod.js" assert { webpackDefer: "true" }; // error
```

Comment:
@Jack-Works the proposal as of _just_ recently the last meeting changed the keyword from `assert` to `with`. I'm thinking we should probably change the syntax right?

Reply:
the keyword change requires parser support. does the parser webpack use support that?

Reply:
I believe you would have to implement a custom acorn plugin similar to what was done with `import()`. Let's continue as normal for now, but I want to ask TC39 members if this syntax is expected to change _anymore_. I want to avoid shipping this with the current name if it has changed or will continue to change. Even though browsers (Chrome) have implemented `assert`.

Reply:
I understand your concern, this is why I have implemented the comment version.
The latest meeting decides the "assert" should become a deprecated feature and encourage people to migrate it. Maybe we can do that in 3 years?
I don't know what will happen in the future so I have no idea if it will change again.

it's surprising that I can modify the parser to support new syntax, maybe I'll investigate this in the future

Reply:
My expectation (as a proposal co-champion) is that the syntax will not change anymore. That's what was signified by re-advancing to Stage 3 (as opposed to continuing to iterate at Stage 2).